### DISCUSSION TITLE
Angular complains: .prose ol[type="A" s] -> Malformed attribute selector: type="A" s]

### DISCUSSION THREAD

**Discussion by danielmoessner** [tailwindlabs/tailwindcss#4557] [Help] [answered]
I've just updated to angular 12 and now it complains about the default prose styles:

```
  .prose ol[type="A" s] -> Malformed attribute selector: type="A" s]
  .prose ol[type="a" s] -> Malformed attribute selector: type="a" s]
  .prose ol[type="I" s] -> Malformed attribute selector: type="I" s]
  .prose ol[type="i" s] -> Malformed attribute selector: type="i" s]
```

How can I disable or remove those style from being generated at all?

I've tried:

```
theme: {
    extend: {
      colors: {
        blue: '#003c4d'
      },
      typography: {
        DEFAULT: {
          'ol[type="A" s]': false,
          'ol[type="a" s]': false,
          'ol[type="I" s]': false,
          'ol[type="i" s]': false,
        }
      }
    },
  },
```

but that doesn't seem to work.

**Comment by VicGUTT** [comment, marked as answer]
Hmm... I'm not sure but shouldn't it be:

```js
{
    typography: {
        DEFAULT: {
            css: {
                'ol[type="A" s]': false,
                'ol[type="a" s]': false,
                'ol[type="I" s]': false,
                'ol[type="i" s]': false,
            },
        },
    },
}
```

🤔

**Reply by danielmoessner** [reply]
Yess, that's it thank you very much.

 Don't know how I missed the css.

**Comment by sebastienbarre** [comment]
I'm experiencing this as well, this seems new and is only triggered at build time (for me).

My gut feeling is that you are using @tailwindcss/typography
You can solve this by reverting to 0.4.0, I just tested it:
```
yarn add "@tailwindcss/typography@v0.4.0"
```

@adamwathan @neupauer @RobinMalfait @brennanangel : I'm not entirely sure what's going on here, but the change introduced in [Release v0.4.1 · tailwindlabs/tailwindcss-typography](https://github.com/tailwindlabs/tailwindcss-typography/releases/tag/v0.4.1), which addressed [Fix ordered list style type `s` modifier position by neupauer · Pull Request #137 · tailwindlabs/tailwindcss-typography](https://github.com/tailwindlabs/tailwindcss-typography/pull/137) introduced this strange artefact. It's not breaking the build per se, but creates a huge, 64K+ warning output (mostly containing the unoptimized, unpruned Tailwind css stylesheet), and we noticed it because our prod machine was choking creating it.

Thanks

**Reply by adamwathan** [reply]
See response here:

https://github.com/tailwindlabs/tailwindcss/discussions/4722#discussioncomment-925774

**Comment by asimpletune** [comment]
I saw this myself in an issue with using tailwindcss/typography + postcss in Jekyll, and filed an issue with Jekyll's sass converter plugin here https://github.com/jekyll/jekyll-sass-converter/issues/120

For me, dropping down to 0.3.1 (for typography) was necessary.
